File: easy/7. The BlackJack with conditions/BlackJack.java
import java.util.Scanner

class BlackJack {
	public static void main(String[] arguments) {
		Scanner scanner = new Scanner(System.in);

		int card1 = scanner.nextInt();
		card1 = (card1 == 11) ? 1 : card1;
		int card2 = scanner.nextInt();
		card2 = (card2 == 11) ? 1 : card2;
		int blackjack = card1 + card2;

		System.out.println(blackjack > 21 ? 0 : blackjack);
	}
}
		

File: easy/7. The BlackJack with conditions/blackjack.c
#include <stdio.h>

int main()
{
	int card1 = 0;
	int card2 = 0;
	scanf("%d %d", &card1, &card2);

	card1 = (card1 == 11) ? 1 : card1;
	card2 = (card2 == 11) ? 1 : card2;

	printf("%d", card1 + card2 > 21 ? 0 : card1 + card2);

	return 0;
}